vignettes/polars.Rmd

```diff
@@ -51,88 +59,106 @@ dependencies other than R itself. This makes it very fast and lightweight to
 install, and so **polars** can immediately be used to tackle your big (or
 small!) data wrangling tasks.
 
-The rest of this vignette contains some simple examples to get you going. Note
-that the **polars** syntax and expressions are (deliberately) as close to the
-Python implementation as possible, so you can always refer to the
-[(py)polars book](https://pola-rs.github.io/polars-book/user-guide/) for more 
-ideas. Just remember to switch out any `.` (Python) for a `$` (R) when chaining
-methods. For example, here are two equivalent lines of code for some
-hypothetical dataset.
+## Documentation and tutorials
+
+The rest of this vignette contains some simple examples to get you going. 
+
+Users can find detailed documentation for all objects, functions, and methods [on the Reference page of this website.](https://rpolars.github.io/reference/index.html) This documentation can also be accessed from the R console using the typical `?` syntax. For example, we will later use the `DataFrame()` constructor function and apply the `groupby()` method to a `DataFrame` object. The documentation for these can be accessed by typing these commands:
+
+```r
+?DataFrame
+?DataFrame_groupby
+```
+
+The [Polars book](https://pola-rs.github.io/polars-book/user-guide/) offers a
+great introduction to the Polars data frame library, with a very large number
+of examples in Python and Rust. The syntax and expressions in the `polars`
+package for R are (deliberately) as close to the Python implementation as
+possible, so you can always refer to the [polars
+book](https://pola-rs.github.io/polars-book/user-guide/) for more ideas. Just
+remember to switch out any `.` (Python) for a `$` (R) when chaining methods.
+For example, here are two equivalent lines of code for some hypothetical
+dataset.
 
 ```
 # Python
-df.select([pl.col(["A", "B"]).sum()]))
+df.group_by("id").mean()
 
 # R
-df$select(pl$col(c("A", "B"))$sum())
+df$groupby("id")$mean()
 ```
```

Comment:
Could you please split the code block between Python and R and set the info string?

like

````rmd
```python
df.method
```

```r
df$method
```
````

Reply:
d5c256fdfa359a403bcce662a7cec7ff04b1de24